src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,64 @@
+import { useContext, useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { PeopleContext } from '../stores/PeopleContext';
+
+export const PeoplePage = () => {
+  const {
+    people, isLoading, loadingError, filterPeople,
+  } = useContext(PeopleContext);
```

Comment:
why do you need Context if you use it only in one component. Please read [the article](https://blog.logrocket.com/pitfalls-of-overusing-react-context/)